src/components/LocaleSwitcher.tsx

```diff
@@ -1,47 +1,46 @@
 import classNames from 'classnames'
-import i18n from '../i18n.ts'
-import { useTranslation } from 'react-i18next'
 import { useRevalidator } from 'react-router-dom'
+import { useTolgee, useTranslate } from '@tolgee/react'
 
-type LanguageProps = {
-  nativeName: string
-}
-
-type Languages = {
-  [key: string]: LanguageProps
-}
-
-const languages: Languages = {
-  en: { nativeName: 'English' },
-  es: { nativeName: 'Español' },
-}
+const languages = [
+  ['en', 'English'],
```

Comment:
Does this `en` need to be made something like `en-US` perhaps?

Reply:
No, the key in tolgee is `en`.

Reply:
I updated it in tolgee and in the code 👍 . I think there's a chance en-US will match better with detected browser settings.